pkg/apb/deprovision.go

```diff
@@ -51,10 +51,12 @@ func Deprovision(
 	}
 
 	// Might need to change up this interface to feed in instance ids
+	sm := NewServiceAccountManager(log)
 	executionContext, err := ExecuteApb(
 		"deprovision", clusterConfig, instance.Spec,
 		instance.Context, instance.Parameters, log,
 	)
+	defer sm.DestroyApbSandbox(executionContext, clusterConfig)
```

Comment:
How do we handle the error, if `DestroyApbSandbox` has an error? We can make sure that the log of the error happens in the Destroy function, but maybe make it return nothing to make it more clear how to use it?

Reply:
Unless you want to wrap it in another call just for this area.

Reply:
```
defer func(context ExecutionContenxt config ClusterConfig) {
    err := sm.DestoryApbSandbox(execution, config)
    if err != nil {
         log.Errorf("Error occurred destroying sandbox during deprovision: \n%s", err)
    }
}(executionContext, clusterConfig)
```

Would this suffice?

Reply:
I updated to the `DestroyApbSandbox` to simply log loudly when it fails and return nothing. Let me know if I should go with this method instead.